### PR TITLE
chinese version of README.md

### PR DESCRIPTION
**Pull request by hxGkyle**

### CODE REVIEW

File: docs/README.zh.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+[English](../README.md) | [中文](README.zh.md)
+
+欢迎来到 **Learning Resources Repository**！此仓库旨在为您提供各个领域的精选资源，帮助您的个人和职业发展。无论您是学生、专业人士，还是仅仅对某个领域感兴趣，这里都有丰富的资料可以帮助您提升技能和知识。
+
+## 📚 目录 (Contents)
+
+1. [前端开发 (Frontend Development)](#-frontend-development)
+2. [后端开发 (Backend Development)](#-backend-development)
+3. [个人发展 (Personal Development)](#-personal-development)
```
Comment (Charity10) [inline docs/README.zh.md:11]: There should be a slight change in the links. None of the links in the Chinese version are working correctly; most lead to 404 error page.

Try figuring it out yourself and please feel free to reach out if you want my help. 
ps: I don't want to spoon-feed the corrections to you.😉

File: docs/README.zh.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+---
+
+## 🌐 前端开发 (Frontend Development)
+
+探索从初级到高级的前端开发资源，包括HTML、CSS、JavaScript等方面的教程、课程和书籍。
+
+- [**初级 (Beginner Level)**](./FrontendDevelopment/Frontend.md/#-beginner-level)
```
Comment (Charity10) [inline docs/README.zh.md:19]: same with the links here and also in other sections too

**Review by Charity10 — CHANGES_REQUESTED** [review]

**Comment by hxGkyle** [comment]
solved. 
let me know if there needs any extra help

**Comment by Charity10** [comment]
This is good to go👏
but will you be updating the Chinese readme version cos the original Readme will keep on changing as new resources are added

**Comment by hxGkyle** [comment]
i could sync it around per mouth, when i see there is a change. 
and can u merge the request? i'm doing it for school project and it's about to due

**Comment by Charity10** [comment]
> i could sync it around per mouth, when i see there is a change. and can u merge the request? i'm doing it for school project and it's about to due

I will merge it.
The note in the Chinese version said the doc was translated in 2014/10/20 
was that on purpose?
if it wasn't you can change it and send another pull request

**Comment by hxGkyle** [comment]
> > i could sync it around per mouth, when i see there is a change. and can u merge the request? i'm doing it for school project and it's about to due
> 
> I will merge it. The note in the Chinese version said the doc was translated in 2014/10/20 was that on purpose? if it wasn't you can change it and send another pull request

it is, just in case someone didn't notice the version difference

**Pull request merged**